### ISSUE TITLE
Add documents to house members

### ISSUE THREAD

**Issue by jmprathab** [jmprathab/MyHome-Web#60]
## ⚠️ Is your feature request related to a problem? Please describe

* Add documents to house members

## 💡 Describe the solution you'd like

* Users should be able to upload documents to house members

**Comment by weezykon** [comment]
Hello please is this feature still available?
I want to work on it

**Comment by jmprathab** [comment]
@weezykon Yes. The future is yet to be implemented. I have assigned it to you. Thanks for the interest!

P.S: There is an effort going on to redesign the frontend. If you are interested please let us know. Many tasks are pending to be completed.

**Comment by weezykon** [comment]
Okay, so do you intend using cloudinary as a third party media management?

**Comment by jmprathab** [comment]
> Okay, so do you intend using cloudinary as a third party media management, or do you have anyone in mind?

We do have a backend server implemented github.com/jmprathab/MyHome. It already has the document management implemented. This issue involves integrating frontend with the backend.

**Comment by weezykon** [comment]
I noticed, I have not setup the backend by the way. Can I change the login and signup interface? Can you assign that to me?

**Comment by jmprathab** [comment]
> I noticed, I have not setup the backend by the way. Can I change the login and signup interface? Can you assign that to me?

Login and SignUp interfaces were already redesigned. If you are looking to work on some other issue in the meantime, please hold on for a few more hours. I will create new issues to work on.

**Comment by weezykon** [comment]
Okay that's good, but this is the login page.![Uploading 94922670...]![Uploading 94922671...]

**Comment by jmprathab** [comment]
Did you try out the login page from [rebranding](/jmprathab/MyHome-Web/tree/rebranding) branch? I recommend you to try it out. That has the redesigned interface.

**Comment by weezykon** [comment]
Okay thanks chief

**Comment by xshirl** [comment]
I would like to help redesign the front end, if that's ongoing!

**Comment by mslowiak** [comment]
@xshirl 
Can you join our discord? We can catch up there :)

**Comment by harsharay** [comment]
I want to contribute. How can I do that

**Comment by mslowiak** [comment]
@Tobi406 
Can you help here? :)

**Comment by elvinuchendu** [comment]
Hi, do you still need help with the frontend?

**Comment by akshay2742** [comment]
Hi, I'll be more happy to help you out with the frontend, is the issue still exists?

**Comment by dharmesh1999** [comment]
I am also want to help on frontend part.

**Comment by SesisnandoLRNeto** [comment]
Hi, does the project still need some help on the frontend? I would be happy to help

**Comment by vikram-8290** [comment]
hey @jmprathab you still need help in redesigning the frontend ,i am happy to help if you need

**Comment by dcode333** [comment]
@jmprathab you still need help in redesigning the frontend ,i am happy to help if you need

**Comment by abhi-shek01** [comment]
Hi, is there any feature left to be implemented?

**Comment by Vamshi-Birru** [comment]
Hi, does this project still need some help? I would be happy to help

**Comment by chandra-pro** [comment]
@jmprathab I am interested to work in the project and I want to contribute to it.Let me know if you have any good issue

**Comment by zenith-78** [comment]
@jmprathab I am intrested to contribute in your project please let me know  if there is any documentation related work or any where else that I cancontribute

**Comment by divya10mishra** [comment]
@jmprathab Interested to contribute please mark the issues and documentation

**Comment by sheethalpatila** [comment]
Hi @jmprathab and team,

I’m [Sheethal Patil A], a [MERN fullstack developer] with experience in the MERN stack (MongoDB, Express, React, Node.js). I’ve been looking for opportunities to contribute to open-source projects, and I’m really impressed by the work you’re doing with this project.

I’d love to help with this issue [ https://github.com/jmprathab/MyHome-Web/issues/60) ], as it seems to align well with my skills in [React and Node.js]. I’ve taken a look at the code and I’m ready to dive in. Is there anything specific you’d like me to focus on, or any additional context I should be aware of before getting started?

If there are other areas where you need help, please let me know! I’m eager to contribute and learn more.

Looking forward to your guidance!

Best regards,  
[Sheethal Patil A]
 [[Profile](https://github.com/sheethalpatila)]
